### ISSUE TITLE
Eachine T-28 Model

### ISSUE THREAD

**Issue by Spring7186** [bd4/edgetx-models#2]
I ask for advice from anyone who can kindly help me: which file should I download for an Eachine T-28 to fly with a Radiomaster Zorro with EdgeTx version 2.10.4 installed? I suppose it will also have to be modified, having been written for the Radiomaster Boxer, I will take care of it later. Many thanks in advance for your help.

**Comment by Spring7186** [comment]
ok, I'll answer by myself on my question: for those who want to program their own OpenTX/EdgeTx radio you can follow this video

https://youtu.be/jW5lsxxS2z0?feature=shared

It's in Italian language, so you can sucks 😄 or just use the live YouTube translator. 
Hope this will be useful for you, cheers and happy fly!

**Comment by bd4** [comment]
Unfortunately there is no seamless way to translate between radios. I have copied models over from Boxer to TX16S and to Zorro, using the EdgeTX Companion software. It has profiles for all the supported radios, and can read the configuration from the radio. It goes like this:

1. Choose target radio profile, e.g. Zorro in this case
2. Go to file -> open and choose the model file you want to convert. You will get a warning that the radio does not match
3. Go through the configuration and check every physical switch and make it what you want on the new radio (including Inputs, Flight Modes, Mixes, Outputs, Logical Switches, and Special Functions).

Another option is to choose the source radio profile, e.g. Boxer in this case, open the model file and look at the configuration just to get ideas, and then manually set up the new radio.

Glad you found a helpful video. RC Video Reviews and Painless 360 on youtube both have excellent content in english on OpenTX and EdgeTX.

**Comment by Spring7186** [comment]
Yes, I'm also conscious of difficult arranging the same model on different radio, for this reason I indicated the video who is related to the same option we can find in Open/Edge-TX radio, hoping this should simplify the process
